index.js

```diff
@@ -1173,7 +1175,7 @@
     }
   }
 
-  //# test :: (Array Type, Type, a) -> Boolean
+  //# test :: Array Type -> Type -> a -> Boolean
```

Comment:
The internal `test` function is not curried, but the exported `test` function is:

```javascript
return {
  // ...
  test: def('test', {}, [Array_(Type), Type, Any, Boolean_], test),
  // ...
};
```

Having this function curried is very convenient. I can imagine partially applying it to fix the environment:

```javascript
//    env :: Array Type
const env = ...;

//    test :: Type -> a -> Boolean
const test = $.test(env);
```